### ISSUE TITLE
Pseudovitenskap som sjanger

### ISSUE THREAD

**Issue by kyrretl** [realfagstermer/realfagstermer#314]
Det er en del bøker i 42-samlingen som *er* pseudovitenskap, f.eks. Michael Behes 'Darwin's Black Box' og William A. Dembskis 'No Free Lunch' (de er begge kreasjonismebøker). Men uinder samme emneord 'Pseudovitenskap' har vi mange bra bøker som er *om* pseudovitenskap som Ben Goldacres 'Bad Science'. Foreslår at vi bruker Pseudovitenskap som sjanger om de bøkene som er pseudovitenskap og emneordet om bøkene som beskriver det.

Kyrre

P.S. syns vi burde bruke sjanger i alle tilfeller der det er tvil om det er *om* eller *er*

**Comment by mittinatten** [comment]
+1

**Comment by BioHeidi** [comment]
Men, @bibliomari - vi (utvidet begrep av vi) diskuterte nettopp i emnered om det var noen grunn til å skille mellom statistikk som form eller emne, og standarder som form eller emne. Siden alle form-ord også kan brukes som emne kan vi ikke se på en post om termen som er satt på er form eller emne? Eller misforsto jeg feil vei?

**Comment by danmichaelo** [comment]
> Siden alle form-ord også kan brukes som emne kan vi ikke se på en post om termen som er satt på er form eller emne? 

Jo, det kan vi. Så hvis vi gjør om Pseudovitenskap til form-ord vil det kunne brukes som både form (655) og emne (687). Jeg oppfatter det som at det er det Kyrre foreslår.

> Og, jeg minnes en diskusjon som omhandlet emne behandlet i en bestemt sjanger, eks. geologi i science fiction etc. For å forkludre det mer.

Deeet tror jeg vi bør ta i egen tråd ;)

**Comment by Hypsibius** [comment]
Reit bortsett frå at dette også bør markerast gjennom Dewey-nummer. Intelligent design og anna kreasjonisme skal på 200-talet: 
![intelligent design](https://cloud.githubusercontent.com/assets/9350370/9688580/4b334f0e-5330-11e5-9ba0-0c692fa52af6.PNG)
